sound/hybrid_font.h

```diff
@@ -147,11 +149,14 @@ class HybridFont : public SaberBase {
           swing_player_->set_fade_time(swing_player_->length() - swing_player_->pos());
           swing_player_->FadeAndStop();
           swing_player_.Free();
-          swing_player_ = PlayPolyphonic(&swng);
         }
       }
-      else if (!swing_player_) {
-        swing_player_ = PlayPolyphonic(&swng);
+      if (!swing_player_) {
+        if (fabs(swing_acceleration_) > config_.ProffieOSSlashAccelerationThreshold && slsh.files_found()) {
+          swing_player_ = PlayPolyphonic(&slsh);
+        } else if (!swinging_){
```

Comment:
adding the !swinging_ check is unrelated to the slash implementation, and I'm not sure it's correct either.

Reply:
Yes, I found a bug in the swing code while I was testing.  This check is to ensure that swing files don't play more than once during a long sweeping swing.

Reply:
That's cool, but bugfixes should really go in separate pull requests.

Reply:
This structure has to change as part of the migration of the speed related code into the startswing function.